recipes/rsem/Makefile.new

```diff
@@ -0,0 +1,158 @@
+CC = g++
+CFLAGS = -Wall -c -I.
```

Comment:
I guess both Makefiles, the new and the orig, are not needed in this PR as you have the patch?